monkey/tests/unit_tests/infection_monkey/agent_event_handlers/test_plugin_event_forwarder.py

```diff
@@ -0,0 +1,62 @@
+from ipaddress import IPv4Address
+from multiprocessing import Queue
+from time import sleep
+from typing import FrozenSet, Union
+from unittest.mock import MagicMock
+from uuid import UUID
+
+import pytest
+
+from common.agent_events import AbstractAgentEvent
+from common.event_queue import IAgentEventQueue
+from infection_monkey.agent_event_handlers import PluginEventForwarder
+
+
+class FakeEvent(AbstractAgentEvent):
```

Comment:
"Fake" has a very specific meaning. See https://martinfowler.com/bliki/TestDouble.html

This needs a better name.